typing/typecore.ml

```diff
@@ -720,8 +721,9 @@ end) = struct
                                                 paths, false, expansion))
           end;
           lbl
-        with Not_found -> try
-          let lbl = lookup_from_type env tpath usage lid in
+        | exception Not_found ->
+        match lookup_from_type env tpath usage lid with
+        | lbl ->
```

Comment:
I summon @Drup to fight this battle for me.

Reply:
Taratata !

> "Justification-strengths should match change-importance." The strength of the justification required for a code-change decision is proportional to the importance of the change. A change with a large impact should be very strongly justified in discussions with reviewers, but for a nitpick-level change the author needs few justifications (possibly just personal preference) to apply or ignore a change suggestion.

This PR is joint work with @Octachron and @garrigue and both saw the code style in real time. OCtachron didn't say anything, and @garrigue approved.

Reply:
Please.

> This does not mean that whitespace or comment changes can be made completely on whims, as different whitespace/comment changes have different impact and importance.

You're trying to sneak in a new way to indent things, and:
- everyone can already predict that you'll try to leverage it in other places later ("well if it was fine for that PR on typecore, why wouldn't it be nice for this other thing here...")
- it *has* an impact on readability.

You seem to have contradictory desires @gasche: on the one hand you want your refactor PRs to be merged quickly, on the other hand you insist on adding friction in every one of them.

:)

Reply:
There's an `.ocp-indent` file in the project isn't that supposed to be applied ? At least I did in [the past](https://github.com/ocaml/ocaml/pull/8834#discussion_r307978072) even if that was not to my liking.